Sources/HTML/Elements.swift

```diff
@@ -226,6 +226,10 @@ public func style(_ css: String) -> Node {
   return node("style", [], [.text(EncodedString(css))])
 }
 
+public func svg(_ attribs: [Attribute], _ svg: String) -> Node {
+    return node("svg", attribs, [.text(EncodedString(svg))])
```

Comment:
purposely so tho! they are allowed to be unencoded in those tags in particular.

was thinking we could have an `internal` function `safeString` that is just the initializer of `EncodedString`

the cool thing still is that outside this framework no one is allowed to insert unencoded text. but this framework does have to enforce the rules and know what it is doing